newsfragment/3401.bugfix

```diff
@@ -0,0 +1,2 @@
+Fix | On Android it seems to be impossible to view the complete description of a Space (without dev tools) (#3386)
+Fix | Suggest Rooms, Show a detailed view of the room on click (#3406)
```

Comment:
Thanks. The text should not contains the issue number. In this case, you should create 2 files:
`3386.bugfix` and `3406.bugfix`.
Anyway, I will manage it, no worries